### PR TITLE
Start pv60 sync with 10sec delay

### PR DESCRIPTION
**Pull request by arkpar**

### CODE REVIEW

File: libethereum/EthereumHost.h [hunk 1]
```diff
@@ -138,6 +138,7 @@ class EthereumHost: public p2p::HostCapability<EthereumPeer>, Worker
 	mutable Mutex x_transactions;
 	DownloadMan m_man;
 	std::unique_ptr<BlockChainSync> m_sync;
+	std::atomic<time_t> m_syncStart = { 0 };
```
Comment (gavofyork) [inline libethereum/EthereumHost.h]: prefer usage of `chrono`...

File: libethereum/EthereumHost.cpp [hunk 2]
```diff
@@ -115,6 +116,21 @@ void EthereumHost::doWork()
 
 	foreachPeer([](std::shared_ptr<EthereumPeer> _p) { _p->tick(); return true; });
 
+	if (m_syncStart)
+	{
+		Guard l(x_sync);
+		if (!m_sync)
+		{
+			time_t now = std::chrono::system_clock::to_time_t(chrono::system_clock::now());
+			if ((now - m_syncStart) > 10)
```
Comment (gavofyork) [inline libethereum/EthereumHost.cpp]: reason for `(` `)`s?

File: libethereum/EthereumHost.cpp [hunk 3]
```diff
@@ -115,6 +116,21 @@ void EthereumHost::doWork()
 
 	foreachPeer([](std::shared_ptr<EthereumPeer> _p) { _p->tick(); return true; });
 
+	if (m_syncStart)
+	{
+		Guard l(x_sync);
```
Comment (gavofyork) [inline libethereum/EthereumHost.cpp]: consider `DEV_GUARDED(x_sync)` to avoid additional `{` `}`s

File: libethereum/EthereumHost.h (hunk 1 above)
Comment (arkpar) [inline libethereum/EthereumHost.h]: would require unnecessary mutex locking

**Pull request merged**